projects/steering/test/test_steering_can.c

```diff
@@ -0,0 +1,63 @@
+#include "can.h"
+#include "can_msg_defs.h"
+#include "event_queue.h"
+#include "exported_enums.h"
+#include "gpio.h"
+#include "gpio_it.h"
+#include "interrupt_def.h"
+#include "ms_test_helpers.h"
+#include "soft_timer.h"
+#include "status.h"
+#include "steering_can.h"
+#include "steering_digital_input.h"
+#include "steering_events.h"
+#include "test_helpers.h"
+
+#define REGEN_BRAKING_TOGGLE_CAN \
+  { .id = 12, .data = 0 }
+
+static CanStorage s_can_storage;
+
+static CanSettings can_settings = { .device_id = SYSTEM_CAN_DEVICE_STEERING,
+                                    .bitrate = CAN_HW_BITRATE_500KBPS,
+                                    .rx_event = STEERING_CAN_EVENT_RX,
+                                    .tx_event = STEERING_CAN_EVENT_TX,
+                                    .fault_event = STEERING_CAN_FAULT,
+                                    .tx = { GPIO_PORT_A, 12 },
+                                    .rx = { GPIO_PORT_A, 11 },
+                                    .loopback = true };
+
+static int count = 0;
+
+StatusCode prv_test_regen_toggle_rx_cb_handler(const CanMessage *msg, void *context,
+                                               CanAckStatus *ack_reply) {
+  TEST_ASSERT_EQUAL(SYSTEM_CAN_MESSAGE_REGEN_BRAKING_TOGGLE_REQUEST, msg->msg_id);
+  count++;
+  return STATUS_CODE_OK;
+}
+
+void setup_test(void) {
+  gpio_init();
+  interrupt_init();
+  event_queue_init();
+  gpio_it_init();
+  soft_timer_init();
+  TEST_ASSERT_OK(steering_digital_input_init());
+  TEST_ASSERT_OK(can_init(&s_can_storage, &can_settings));
+}
+
+void test_steering_digital_input_cc_toggle() {
```

Comment:
Nitpick: this parameter list should be `(void)` rather than `()`. Also I think you forgot to rename the test? This isn't testing steering_digital_input nor is it testing cruise control toggling...